docs/rules/no-test-return-statement.md

```diff
@@ -0,0 +1,47 @@
+# Disallow returning from tests (no-test-return-statement)
+
+Tests in Jest are void and cannot return values.
```

Comment:
"should be"? Seeing as you can return a promise, and that's unlikely to change (I don't think it's possible to differentiate between an `async` function and a promise-returning one (at least not if it's transpiled))